pkg/cmd/server/origin/handlers.go

```diff
@@ -21,30 +21,32 @@ import (
 	"github.com/openshift/origin/pkg/authorization/authorizer"
 )
 
+// TODO once we have a MuxHelper we will not need to hardcode this list of paths
+// RootPaths are paths we server on from root
+var RootPaths kapi.RootPaths = kapi.RootPaths{Paths: []string{
+	"/api",
+	"/api/v1beta3",
```

Comment:
Probability of @deads2k asking for another integration test any given day... rising.